index.html

```diff
@@ -155,57 +159,62 @@ <h3 class="designation">Senior Mentor</h3>
 		</section>
 
 		<section class="contact">
-			
-			<div class="main-register">
-				<h1 class="register-heading">Register Your Self</h1>
-				<p class="register-para">Sed ut perspiciatis, unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.</p>
-
-				<div class="sub-register">
-					<img src="images/be-mentor.jpg" alt="be-mentor" class="register-img">
-					<h2 class="register-text">I want to be a mentor</h2>
-				</div>
-
-				<div class="sub-register">
-					<img src="images/want-mentor.jpg" alt="want-mentor" class="register-img">
-					<h2 class="register-text">I want a mentor</h2>
-				</div>
-
-				<div class="sub-register">
-					<img src="images/general.jpg" alt="general" class="register-img">
-					<h2 class="register-text">General Qustions</h2>
-				</div>
-
-            </div>
-
-            <div class="form">
-            	
-            	<h1 class="form-heading">Fill The Form</h1>
-
-            	<form method="post" action="#">
-
-            		<input type="checkbox" name="">Register as a Mentee
-            		<input type="checkbox" name="">Register as a Mentor <br>
 
-            		<label class="form-label">Name*</label><br>
-            		<input type="text" name="" class="form-input"><br>
-
-            		<label class="form-label">Email*</label><br>
-            		<input type="text" name="" class="form-input"><br>
-            		
-            		<label class="form-label">Job Title*</label><br>
-            		<input type="text" name="" class="form-input"><br>
-            		
-            		<label class="form-label">Job Description</label><br>
-            		<input type="text" name="" class="form-input"><br>
-
-            		<label class="form-label">Motivation to Register</label><br>
-            		<input type="text" name="" class="form-input"><br>
-                    
-                    <input type="submit" name="" class="register-button">
-            		
-            	</form>
-            
-            </div>
+			<div class="contact-flex">
+			
+				<div class="main-register">
```

Comment:
All these should go more to the left, so that they are just 2 tabs indented from `<div class="contact-flex">`